src/style.css

```
@@ -0,0 +1,78 @@
+html {
+  --border-color: #e1e7ed;
+  --text-color: #3d4e61;
+
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "AvenirBook";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+@font-face {
+  font-family: "AvenirHeavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.search {
+  align-items: center;
+  display: flex;
+
+  margin: 20px 0;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  max-width: 100%;
+  height: 68px;
+  padding: 0 9px;
+
+  border: 1px solid var(--border-color);
+  border-radius: 4px;
+  background-color: #fff;
+}
+
+.search:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.search--small {
+  height: 40px;
+  padding: 0 4px;
+}
+
+.search__icon {
+  width: 19px;
+  padding: 0 17px;
+}
+
+.search__icon--small {
+  width: 11px;
+  padding: 0 9px;
+}
+
+.search__input {
+  all: unset;
+
+  width: 100%;
+
+  font-family: AvenirBook, Arial, Helvetica, sans-serif;
```

you can just apply font-family to `html` tag selector and let inputs to inherit it 
```suggestion
  font-family: inherit;
```